src/activitypub/fediverse-bridge.ts

```diff
@@ -36,6 +47,91 @@ export class FediverseBridge {
         );
     }
 
+    private async handlePostCreated(event: PostCreatedEvent) {
+        const post = event.getPost();
+        if (!post.author.isInternal) {
+            return;
+        }
+        const ctx = this.fedifyContextFactory.getFedifyContext();
+        let fedifyObject = null;
+
+        if (post.type === PostType.Note) {
+            if (post.inReplyTo) {
+                return;
```

Comment:
Reply activity is already sent in `reply.ts`, this was sending a duplicate activity.
I have added a PR for moving code for sending reply to Fediverse from reply.ts to FediverseBridge [here](https://github.com/TryGhost/ActivityPub/pull/636).  Kept it separate to avoid too many changes in this one as it's already a big PR.